### PR TITLE
[CAS-557] Visibility for config repository

### PR DESCRIPTION
**Pull request by leandroBorgesFerreira**
[CAS-557](https://stream-io.atlassian.net/browse/CAS-557)

### Description

Changing visibility of `RepositoryHelper.configs` and creating necessary methods. 

### Checklist

- [x] I have signed the [Stream CLA](https://docs.google.com/forms/d/e/1FAIpQLScFKsKkAJI7mhCr7K9rEIOpqIDThrWxuvxnwUq2XkHyG154vQ/viewform) (required)
- [x] PR targets the `develop` branch
- ~Changelog updated with client-facing changes~
- ~New code is covered by unit tests~
- ~Comparison screenshots added for visual changes~
- [x] Reviewers added

### CODE REVIEW

File: stream-chat-android-offline/src/main/java/io/getstream/chat/android/livedata/ChatDomainImpl.kt [hunk 1]
```diff
@@ -754,7 +754,7 @@ internal class ChatDomainImpl internal constructor(
         }
 
         // store the channel configs
-        repos.configs.insert(configs)
+        repos.configChannel(configs)
```
Comment (ogkuzmin) [inline stream-chat-android-offline/src/main/java/io/getstream/chat/android/livedata/ChatDomainImpl.kt:757]: Why not `insertChannelConfig`?

Comment (leandroBorgesFerreira) [reply stream-chat-android-offline/src/main/java/io/getstream/chat/android/livedata/ChatDomainImpl.kt:757]: My though is that this method is used to configure the channel, that fact that the configuration is stored can be extracted away. But no problems with `insertChannelConfig` either, what do you think?

Comment (ogkuzmin) [reply stream-chat-android-offline/src/main/java/io/getstream/chat/android/livedata/ChatDomainImpl.kt:757]: I consider RepositoryHelper just as a database wrapper. ChatDomain is a key for logic. So I think straightforward insert is better than

Comment (leandroBorgesFerreira) [reply stream-chat-android-offline/src/main/java/io/getstream/chat/android/livedata/ChatDomainImpl.kt]: Changed

**Review by ogkuzmin — APPROVED** [review]

**Pull request merged**